libs/coin-evm/src/adapters/ethers.ts

```diff
@@ -19,7 +20,10 @@ export const transactionToEthersTransaction = (tx: EvmTransaction): ethers.Trans
     value: ethers.BigNumber.from(tx.amount.toFixed(0)),
     data: tx.data ? `0x${tx.data.toString("hex")}` : undefined,
     gasLimit: ethers.BigNumber.from(gasLimit.toFixed(0)),
-    nonce: tx.nonce,
+    // When using DEFAULT_NONCE (-1) ethers might break on some methods,
```

Comment:
In what case do we see a -1 nonce (that now becomes max integer value in OP/Arbitrum case) ?

Reply:
Nonce has been set by Alexandre at -1 to be able to recognize a brand new transaction crafted by the live, and and transaction being "replaced by fee" where we already know the nonce we want to use.
This way, once reaching the `prepareForSignOperation` method, we know that if the transaction has "-1" has nonce, we should ask a node for the correct one, otherwise we let the nonce already provided.